tmexp/preprocess.py

```diff
@@ -221,7 +228,7 @@ def feature_extractor(uast_obj: Any) -> Iterator[Tuple[str, str]]:
                 if time.time() - start > bblfsh_timeout - 0.1 and attempt == 0:
                     logger.warn("Babelfish timed out, restarting the container ...")
                     subprocess.call(
-                        ["docker", "restart", bblfsh_container],
+                        ["docker", "restart", bblfsh_host],
                         stdout=open(subprocess.DEVNULL, "wb"),
```

Comment:
Did not notice last time but we don't need open with subprocess.DEVNULL.

Reply:
ah we can just put `stdout=subprocess.DEVNULL` ?

Reply:
I do think so